frontend/packages/core/src/Input/stories/text-field.stories.tsx

```diff
@@ -50,3 +50,22 @@ WithEndAdornment.args = {
   defaultValue: "Search",
   endAdornment: <SearchIcon />,
 };
+
+const autoComplete = value => {
+  return new Promise((resolve, reject) => {
+    resolve({
+      results: [
+        { id: "clutch", label: "" },
```

Comment:
super nit: it would be cool to have these be configurable within the story but that's a nice to have not a requirement

Reply:
oh yeah let me add that